<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Set background color for QMainWindow */
QMainWindow {
    background-color: #F0F0F0;
}

/* Set font and size for QPushButton */
QPushButton {
    font: bold 14px;
    color: #FFFFFF;
    background-color: #007ACC;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

/* Set hover effect for QPushButton */
QPushButton:hover {
    background-color: #009FE3;
}

/* Set font and size for QLineEdit */
QLineEdit {
    font: 12px;
    color: #000000;
    background-color: #FFFFFF;
    border: 1px solid #CCCCCC;
    border-radius: 3px;
    padding: 5px;
}

/* Set font and size for QLabel */
QLabel {
    font: bold 12px;
    color: #000000;
}

/* Set spacing and alignment for labels and line edits */
QLabel, QLineEdit {
    margin: 5px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Read">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Delete">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Insert">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Update">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Value1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Value2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mobile number</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCRUD">
    <property name="title">
     <string>CRUD</string>
    </property>
   </widget>
   <addaction name="menuCRUD"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
